app/controllers/orders_controller.rb

```diff
@@ -30,6 +30,17 @@ def add_order_photos(order)
     end
   end
 
+
+  def add_default_order_status(order)
+    current_status = Status.find_by(name: "Ordered")
+    if current_status
+      order.status = current_status
+    else
+      new_status = Status.create(name: "Ordered")
+      order.status = new_status
+    end
+  end
+
```

Comment:
could this be refactored to "order.status = Status.find_or_create_by(name: "Ordered)"?

(thanks @sgrif !)